Sources/TuistAutomation/Device/DeviceController.swift

```diff
@@ -158,11 +178,24 @@ extension PhysicalDevice {
         case .watchOS: .watchOS
         }
 
+        let transportType: PhysicalDevice.TransportType = switch device.connectionProperties.transportType {
+        case .localNetwork: .wifi
+        case .wired: .usb
+        case .none: .unknown
```

Comment:
Should we map `.none` also to `.none`? It's not an `unknown` case in the end, is it?